src/gobstuf/stuf/brp/request/ingeschrevenpersonen.py

```diff
@@ -51,6 +53,10 @@ class IngeschrevenpersonenFilterStufRequest(IngeschrevenpersonenStufRequest):
         'verblijfplaats__gemeentevaninschrijving': [ArgumentCheck.has_min_length(GEMEENTECODE_LENGTH),
                                                     ArgumentCheck.has_max_length(GEMEENTECODE_LENGTH),
                                                     ArgumentCheck.is_valid_gemeentecode],
+        'verblijfplaats__identificatiecodenummeraanduiding': [
```

Comment:
Maybe add the ArgumentCheck.is_integer check to this parameter? They should all be integers according to stelselpedia